django_editorjs_fields/config.py

```diff
@@ -95,8 +95,7 @@
         '@editorjs/marker',
         '@editorjs/table',
         '@editorjs/underline',
-        'editorjs-paragraph-with-alignment',
-        'editorjs-header-with-alignment'
+        'editorjs-text-alignment-blocktune'
```

Comment:
I don`t use this plugin anymore(another plugin for text alignment in master branch)